src/surge-xt/gui/SurgeGUIEditorValueCallbacks.cpp

```diff
@@ -2478,27 +2478,49 @@ int32_t SurgeGUIEditor::controlModifierClicked(Surge::GUI::IComponentTagValue *c
                                 .filterblock_configuration.val.i == fc_wide)
                         {
                             contextMenu.addSeparator();
-                            auto dt = p->deform_type;
+                            auto dt = p->deform_type & 0x1;
 
                             Surge::Widgets::MenuCenteredBoldLabel::addToMenuAsSectionHeader(
                                 contextMenu, "NOISE GENERATOR MODE");
 
                             contextMenu.addItem(
                                 "Mono", true, dt == NoiseColorChannels::MONO, [this, p]() {
                                     undoManager()->pushParameterChange(p->id, p, p->val);
-                                    update_deform_type(p, NoiseColorChannels::MONO);
+                                    update_deform_type_bit(p, NoiseColorChannels::MONO, 0);
                                     synth->storage.getPatch().isDirty = true;
                                     frame->repaint();
                                 });
                             contextMenu.addItem(
                                 "Stereo", true, dt == NoiseColorChannels::STEREO, [this, p]() {
                                     undoManager()->pushParameterChange(p->id, p, p->val);
-                                    update_deform_type(p, NoiseColorChannels::STEREO);
+                                    update_deform_type_bit(p, NoiseColorChannels::STEREO, 0);
                                     synth->storage.getPatch().isDirty = true;
                                     frame->repaint();
                                 });
                         }
+                        {
+                            contextMenu.addSeparator();
+                            auto dt = (p->deform_type & 0x2) >> 1;
 
+                            Surge::Widgets::MenuCenteredBoldLabel::addToMenuAsSectionHeader(
+                                contextMenu, "NOISE GENERATOR COLOR TYPE");
+
+                            contextMenu.addItem(
+                                "Legacy (Pre-XT 1.4)", true, dt == NoiseColorValue::LEGACY,
```

Comment:
In Sine oscillator we used this verbiage: `Legacy (<1.6.2)`. So over here it would probably make sense to be consistent and say `Legacy (<XT 1.4)`.

Reply:
Done.